nwb_conversion_tools/datainterfaces/ecephys/baserecordingextractorinterface.py

```diff
@@ -112,7 +112,6 @@ def run_conversion(
         use_times: bool = False,
         save_path: OptionalPathType = None,
         overwrite: bool = False,
-        buffer_mb: int = 500,
```

Comment:
This should have been removed with PR #218, but I guess I missed it there. Right now it's a completely unused optional argument, so doesn't throw any error but also doesn't do anything.